### PR TITLE
[srp-client] add lease changing test

### PR DESCRIPTION
**Pull request by simonlingoogle**
Verified that the test can be fixed by https://github.com/openthread/openthread/pull/7281

### CODE REVIEW

File: tests/scripts/thread-cert/node.py [hunk 1]
```diff
@@ -1111,6 +1111,16 @@ def srp_client_get_services(self):
         service_lines = self._expect_command_output()
         return [self._parse_srp_client_service(line) for line in service_lines]
 
+    def srp_client_set_lease_interval(self, leaseinterval: int):
+        cmd = 'srp client leaseinterval %d' % leaseinterval
```
Comment (wgtdkp) [inline tests/scripts/thread-cert/node.py:1115]:
```suggestion
        cmd = f'srp client leaseinterval {leaseinterval}'
```

**Comment by simonlingoogle** [comment]
TIP:
- Need merge https://github.com/openthread/openthread/pull/7281 first.

@wgtdkp @jwhui

**Review by abtink — APPROVED** [review]
LGTM. 👍

**Comment by wgtdkp** [comment]
@simonlingoogle #7281 is merged, please rebase！

**Comment by simonlingoogle** [comment]
> @simonlingoogle #7281 is merged, please rebase！

Actually, I rebased on #7281 days ago but the CI was still failing. 

I have fixed the CI by changing Wireshark configuration in the 2nd commit.

File: tests/scripts/thread-cert/pktverify/consts.py [hunk 2]
```diff
@@ -338,6 +338,25 @@
...
+    'udp.port==53548': 'dns',
+    'udp.port==53552': 'dns',
+    'udp.port==53554': 'dns',
+    'udp.port==53542': 'dns',
+    'udp.port==53545': 'dns',
+    'udp.port==53549': 'dns',
+    'udp.port==53551': 'dns',
+    'udp.port==53553': 'dns',
+    'udp.port==53535': 'dns',
+    'udp.port==53537': 'dns',
```
Comment (wgtdkp) [inline tests/scripts/thread-cert/pktverify/consts.py:359]: sort the entries?

**Review by wgtdkp — APPROVED** [review]
LGTM :+1:

File: tests/scripts/thread-cert/pktverify/consts.py (hunk 2 above)
Comment (simonlingoogle) [reply tests/scripts/thread-cert/pktverify/consts.py]: sorted.

**Review by jwhui — APPROVED** [review]

**Pull request merged**